src/dmd/json.d

```diff
@@ -302,8 +302,7 @@ public:
             //property(name, "impure");
             break;
         case PURE.weak:     return property(name, "weak");
-        case PURE.const_:   return property(name, "const");
-        case PURE.strong:   return property(name, "strong");
+        case PURE.const_:   return property(name, "strong");
```

Comment:
> Is this mismatch deliberate?

You mean "const_" vs "strong"? I can rename the enum if you like. I just kept it as "PURE.const_" to make sure that I'm not missing "PURE.strong" being used with the old meaning.

Reply:
I think `PURE.strong` is a better name, but I'm fine with this as well for this PR, it can easily be renamed later. I just wanted to make sure this wasn't a mistake.